lib/AST/ASTScopeSourceRange.cpp

```diff
@@ -408,20 +408,13 @@ SourceRange ForEachPatternScope::getSourceRangeOfThisASTNode(
 }
 
 SourceRange
-CaseStmtScope::getSourceRangeOfThisASTNode(const bool omitAssertions) const {
-  // The scope of the case statement begins at the first guard expression,
-  // if there is one, and extends to the end of the body.
-  // FIXME: Figure out what to do about multiple pattern bindings. We might
-  // want a more restrictive rule in those cases.
```

Comment:
Is there some new comment that belongs somewhere to replace this one?

Reply:
I've added a comment to CaseStmtScope that I think explains everything.